### PR TITLE
Update contribution guidelines with light mode instructions

### PR DESCRIPTION
**Pull request by sannan06**
**Describe the issue**
There is no documentation for how to handle dark & light mode in developing new components

**Describe the solution**
Update CONTRIBUTING.md with a new section on how to handle dark & light modes

### CODE REVIEW

**Review by kcho9906 — APPROVED** [review]
Huge props to you for sorting out this ticket - documentation is clear on how to cater for future work. Great work!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -154,6 +155,15 @@ Names of project files should:
 Mello uses the BEM Naming structure, which aims to keep CSS codebases structured with a consistent naming convention.
 _For more information, see https://yandex.com/dev/bem/_
 
+### Dark and Light Mode
+
+The default colour scheme for the application is dark mode, and as such all components should have their default stylign consistent with dark mode. However, the application also has support for light mode, and your component/extensions must also have support for light mode. The following steps will add light mode support for your component:
```
Comment (Denijar) [inline CONTRIBUTING.md]: Just noticing something here - when I checkout `main-group2` and run the application, it defaults to light mode even though I have not touched any settings. Is this a bug I am experiencing, or is light mode the new default?

**Review by Denijar — CHANGES_REQUESTED** [review]

**Comment by Denijar** [comment]
I am confused why the checks are not passing. The test that is failing is the one that I thought we fixed in #242

**Comment by kcho9906** [comment]
> I am confused why the checks are not passing. The test that is failing is the one that I thought we fixed in #242

Maybe @sannan06 rebased with main instead of main-group2?

File: CONTRIBUTING.md (hunk 1 above)
Comment (sannan06) [reply CONTRIBUTING.md]: The default is whatever your system is in (i.e. based on if your Mac/pc is in dark/light mode)

**Comment by sannan06** [comment]
> I am confused why the checks are not passing. The test that is failing is the one that I thought we fixed in #242

Resolved; was caused by branching off master instead of new branch

**Review by Denijar — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Denijar) [reply CONTRIBUTING.md]: ah okay - would it make more sense to say this then?

**Pull request merged**